staging/src/k8s.io/client-go/informers/generic.go

```diff
@@ -100,6 +101,10 @@ func (f *sharedInformerFactory) ForResource(resource schema.GroupVersionResource
 	case v1beta1.SchemeGroupVersion.WithResource("validatingwebhookconfigurations"):
 		return &genericInformer{resource: resource.GroupResource(), informer: f.Admissionregistration().V1beta1().ValidatingWebhookConfigurations().Informer()}, nil
 
+		// Group=apiserverinternal, Version=v1alpha1
```

Comment:
`Group=apiserverinternal` is wrong.
I think if you keep the groupName tag, the comment here would be correct.